backend/grants/admin.py

```diff
@@ -317,12 +310,59 @@ class GrantAdmin(ExportMixin, AdminUsersMixin, SearchUsersMixin):
     )
     user_fk = "user_id"
     actions = [
-        send_messages_to_plain,
         send_reply_emails,
         send_grant_reminder_to_waiting_for_confirmation,
         "delete_selected",
     ]
 
+    fieldsets = (
```

Comment:
weird order where "Manage" appears before "The grant"?

Reply:
Because in the admin we are editing this information mostly, so it's better to have this section first in my opinion. And Django automatically will show the `full_name` of the person in bold at the top, so you know what grant you are editing already.

Reply:
if you think about we have kind the same order now in production, without the fieldsets: 
![Screenshot 2023-02-11 at 12 46 07](https://user-images.githubusercontent.com/28976199/218258706-c10c1326-ddb5-481c-8c30-cd6d350852c8.png)

The new admin, you see the full name here: 
![Screenshot 2023-02-11 at 12 45 35](https://user-images.githubusercontent.com/28976199/218258741-e2b4e9ac-8af3-47e9-acd1-6d0dcdf46459.png)

Reply:
The default order in production is a "random" one so I wouldn't necessarily use it as an example of "good UX to avoid mistakes"

If you think your order is ok and works then feel free to leave it as it is